Initial release of home page

tomheller opened the pull request:
Here is the initial release-candidate for the home page. 
I think this can be released now (if you want to, of course) and iterated over in the next couple of days. 

There are still a few things, that do not match the design perfectly: 
* [ ] Font-sizes and weights need to be adjusted.
* [ ] Margins can be unified.
* [ ] Hero section and additional circle animation should be revisited. 

As the homepage contains quite a lot of custom sections, the source code for it is not very well componentised. If we go forth with blog and project pages, as they are indicated by the design, we should be able to generalise a lot more, as there will be multiple pages with the same post-type (archetype).

tomheller committed:
initial hugo setup

project code generated with `hugo new site`

tomheller committed:
initial setup of getcloudnative.io theme

generated with `hugo new theme` command

tomheller committed:
chore(setup): scss asset pipeline

it makes sense to use scss and the asset pipeline event for small projects like this to improve performance through minification

tomheller committed:
chore(setup): add bootstrap css and js from cdn and Lato Google font

tomheller committed:
feat(gloabstyles): Use Lato font on the site

tomheller committed:
feat(navbar): initial version of the navbar

tomheller committed:
feat(herosection): initial styling for hero section without circles

tomheller committed:
feat(quote): add quote shortcode

tomheller committed:
feat(section): layout sections offerings and walk the talk

added button styling overrides

tomheller committed:
chore(structure): move website into root folder

tomheller committed:
feat(testimonial): enhance quote to work with testimonial

tomheller committed:
chore(deployment): add netlify build config

tomheller committed:
feat(footer): add footer to layout partials

tomheller committed:
fix(technologygrid): fix responsive behaviour to work on < 420px screens

tomheller committed:
chore: remove generated assets from tracking

tomheller committed:
fix(hero): add static background to section

metmajer commented:
@tomheller Great work! I've tried publishing this branch on Netlify but failed with the following error: https://discourse.gohugo.io/t/are-asset-pipelines-working-on-netlify-for-anyone/12953/10. Can you please look at the bottom-most answer and help me resolve the issue before we push to trunk?

On top of the items mentioned above, I'd like to have the following items improved according to the Sketch file:

* [x] GCn icon in header and footer are .png, but should be .svg
* [ ] The alignment of the navbar items
* [ ] The height of the hero section
* [ ] Vertical margins across all sections
* [ ] Font colors
* [x] Link sections in footer break in smallest resolution

Other than that, these items should follow next:
* [ ] Carousel with additional testimonials
* [ ] Language selector

tomheller committed:
chore(structure): remove theme and move files to root project

tomheller committed:
fix(netlify): fix(netlify): add branch preview environment config

Based on the instructions given on https://gohugo.io/hosting-and-deployment/hosting-on-netlify/

tomheller committed:
fix(config): remove theme configuration

tomheller committed:
fix(styles): removed sass and moved to css

Netlify depoyments currently do not support hugo/extended version which would be necessary for sass pipeline to work.

tomheller commented:
@metmajer Netlify deployments are working now. Netlify is currently not supporting hugo extended version, which is necessary to process sass pipeline. I have removed the sass pipeline in favour of css. 

I agree on the items that you have specified. Would it make sense to move these items into [github issues](https://github.com/getcloudnative/website/issues), and create a [github kanban project](https://github.com/getcloudnative/website/projects)? That way we would not miss any items which are stated during the discussions in PRs.

tomheller committed:
feat(logo): exchange png logo with svg

tomheller committed:
fix(footer): links no longer break on mobile

metmajer commented:
I'll create the issues and the board asap. Thanks!
